exporter/dynatraceexporter/README.md

```diff
@@ -202,3 +240,10 @@ User should calculate this as `num_seconds * requests_per_second` where:
 - `requests_per_second` is the average number of requests per seconds.
 
 Default: `5000`
+
+### tags (Optional)
+
+**Deprecated: Please use [default_dimensions](#default_dimensions-optional)**
+
+Tags are included as dimensions on all exported metrics.
+Tags must be in the `key=value` dimension format specified by the [metrics ingestion protocol](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/metric-ingestion-protocol/).
```

Comment:
```suggestion
```
I think it's fine to not document the legacy option but to point users in the right direction if they try to figure out where it's gone.